lib/susedistribution.pm

```diff
@@ -371,7 +371,7 @@ sub init_consoles {
                 password => get_var('VIRSH_GUEST_PASSWORD')});
     }
 
-    if (check_var('BACKEND', 'ikvm') || check_var('BACKEND', 'ipmi')) {
+    if (check_var('BACKEND', 'ikvm') || check_var('BACKEND', 'ipmi') || check_var('BACKEND', 'spvm')) {
```

Comment:
I have the feeling we do not want to check on the specific backends but let the backend define some capability which we can check.

Reply:
This is definitely out of scope for this ticket and should be handled in a separate poo ticket.